### DISCUSSION TITLE
Wrapping cols on small screens

### DISCUSSION THREAD

**Discussion by AniaKru95** [vuetifyjs/vuetify#16088] [Q&A] [answered]
Hi, 
I have a problem with wrapping columns on small screens using grid system.

https://codepen.io/AniaKru95/pen/dyKRVWb

In above example, if i decrease screen width I would expect that gray column should be below pink column. 
If I set md=12 on pink and gray columns it works as expected, but i would like to wrap columns only on small screens.

I'd appreciate it for any help.

**Comment by jacekkarczmarczyk** [comment, marked as answer]
Like this? https://codepen.io/jkarczm/pen/xxzrXYN

**Reply by AniaKru95** [reply]
Oh, I see - "cols=12" prop is necessary to work as expected.
Thanks for your help :)
